### DISCUSSION TITLE
[QUESTION] - How to detect if the face detected on webcam is real or from photo graph?

### DISCUSSION THREAD

**Discussion by sujit-baniya** [vladmandic/human#363] [Q&A]
Hi I was wondering if it's possible to detect if the face detected on webcam is real and live and not a photo graph of a person. I was trying this library for KYC Verification and detect real live video on webcam and identify if the persona has placed a photo graph

**Comment by vladmandic** [comment]
i'm not sure what to answer here that hasn't been asked and answered before? see (long) thread in #206
